src/Countries/US/Oregon/Eugene/Eugene.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\Oregon\Eugene;
```

Comment:
there should be a `EugeneEmployer` as well shouldn't there?

Reply:
That's a different ticket. https://spurjob.atlassian.net/browse/ACC-435